fake-api.js

```diff
@@ -31,15 +35,92 @@ function fakeApiMiddleWare (req, res, next) {
   next();
 }
 
-app.use('/', fakeApiMiddleWare );
-app.use('/', express.static('test/fake-api-fixtures', { index: 'index.json' }));
+const jsonParser = bodyParser.json();
+
+app.use('/', fakeApiMiddleWare);
+
+app.get('/collections', (req, res) => {
+  res.send(collectionsJson);
```

Comment:
Does this work across test runs? Meaning if your test to delete a collection comes is the last test in your Cypress run and then you re-run the Cypress collection tests, I would think this list will still have that collection missing from the list.

To resolve this you'll have to update to the fake API storage method in #500